tests/sip-sdp/sdp.syn

```diff
@@ -29,5 +29,44 @@ z=2882844526 -1h 2898848070 0\x0d
 k=prompt\x0d
 a=sendrecv\x0d
 m=audio 30000 RTP/AVP 0 8 97 2 3\x0d
+i=media info\x0d
+c=IN IP4 192.168.1.2\x0d
+k=prompt\x0d
+a=rtpmap:0 pcmu/8000\x0d\x0a";);
+default < (content:"SIP/2.0 200 Ok\x0d
+Via: SIP/2.0/UDP 192.168.1.2:5060;branch=z9hG4bKnp104984053-44ce4a41192.168.1.2;rport\x0d
+From: \"arik\" <sip:816666@voip.brurjula.net>;tag=6433ef9\x0d
+To: <sip:97239287044@voip.brujula.net>\x0d
+Call-ID: 105090259-446faf7a@192.168.1.2\x0d
+CSeq: 1 INVITE\x0d
+User-Agent: Nero SIPPS IP Phone Version 2.0.51.16\x0d
+Expires: 120\x0d
+Accept: application/sdp\x0d
+Content-Type: application/sdp\x0d
+Content-Length: 272\x0d
+Contact: <sip:816666@192.168.1.2>\x0d
+Max-Forwards: 70\x0d
+Allow: INVITE, ACK, CANCEL, BYE, REFER, OPTIONS, NOTIFY, INFO\x0d
```

Comment:
nit : do you have a pcap that is real traffic captured from real client and server (not synthetic one ?)